kartograf/collectors/parse.py

```diff
@@ -60,7 +60,9 @@ def parse_routeviews_pfx2as(context):
             prefix = parse_pfx(prefix)
             asn = asn.upper().rstrip('\n')
 
-            if not prefix or is_bogon_pfx(prefix) or is_bogon_asn(asn):
+            if not prefix:
+                continue
```

Comment:
Hm, unsure if we shouldn't print something to debug in this case since there might be some issue that we should fix. Something like "couldn't parse a prefix from {line}" or so?

Reply:
(And of course the other prefix parse too)